src/speech.js

```diff
@@ -1,97 +1,100 @@
 const speech = {
     launch: {
-        welcome: 'Welcome to The Guardian! You can ask me for news, opinions, reviews and sport. ',
-        reprompt: 'Sorry, what would you like to hear?'
+        welcome_1: `Welcome to The Guardian! Let's get you started.
+        You can ask me for help at any time by simply saying, help.
+        You can ask me for news, opinions, reviews, sport and podcasts.
+        What would you like to hear?`,
+        welcome_2: `Welcome back to The Guardian! You can ask me for news, reviews, sport and podcasts.
+        What would you like to hear?`,
+        welcome_3: `Welcome to the Guardian! What would you like to hear?`,
+        welcome_3_long_time: `Welcome back to the Guardian! You can ask me for news, reviews, sport and podcasts.
+        What would you like to hear?`,
+        reprompt: "Sorry, what would you like to hear?"
     },
     news: {
-        explainer: `'OK great. I can give you more than just the headlines. For example, ask: give me the sport headlines.
-        If you want me to read you stories on a topic. Ask: give me the latest on Brexit.
-        And finally, if you want me to hear the latest opinions on a topic. Say: give me the latest opinions on Brexit. '`,
-        reprompt: 'Sorry, what headlines would you like to hear?'
+        explainer: `Sure. I can give you the headlines, or the latest news or opinions for a topic. What are you interested in?`,
```

Comment:
Use core questions - rather than just "What are you interested in?". Remove "What are you interested in?" as an option